enhancements/node-tuning/containerize-tuned.md

```diff
@@ -0,0 +1,255 @@
+    * Profile selection: Select the right profile for the node via Tuned.spec.recommend
+    * Status reporting: Read tuned logs to report tuning status via Tunes.status
+* [Machine config operator](https://github.com/openshift/machine-config-operator) (MCO) will have a systemd service file to start tuned at boot via podman run using the NTO [image](https://github.com/openshift/cluster-node-tuning-operator/blob/eb35a4ef7ef6072e29faf383078562a70302ee1e/Dockerfile.rhel9#L30):
+
+```yaml
+name: ocp-tuned.service
+enabled: true
+contents: |
+  [Unit]
+  Description=TuneD service from NTO image
+  After=firstboot-osupdate.target systemd-sysctl.service network.target polkit.service
+  # Requires is necessary to start this unit before kubelet-dependencies.target
+  Requires=kubelet-dependencies.target
+  Before=kubelet-dependencies.target
+  ConditionPathExists=/var/lib/ocp-tuned/image.env
+  
+  [Service]
+  Type=exec
+  Restart=on-failure
+  RestartSec=10
+  ExecReload=/bin/pkill --signal HUP --pidfile /run/tuned/tuned.pid
+  #TimeoutStartSec=5
+  ExecStartPre=/bin/bash -c " \
+    mkdir -p /run/tuned "
+  ExecStart=/usr/bin/podman run \
+      --rm \
+      --name openshift-tuned \
+      --privileged \
+      --authfile /var/lib/kubelet/config.json \
+      --net=host \
+      --pid=host \
+      --security-opt label=disable \
+      --volume(s)
+	… 
+      --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","-v=1"]' \
+      $NTO_IMAGE
+  Environment=PODMAN_SYSTEMD_UNIT=%n
+  EnvironmentFile=-/var/lib/ocp-tuned/image.env
+  ExecStop=/usr/bin/podman stop -t 20 openshift-tuned
+  ExecStopPost=/usr/bin/podman rm -f --ignore openshift-tuned
+
+```
+
+This approach would work even for deployments such as HyperShift, openshift-ansible and likely even IBM ROKS; this still wouldn't work for custom deployments with RHEL.
+
+* Prior to the tuned main process, all necessary tuned directories and known profiles will be created and placed on the node.
+
+* This containerized approach requires having a path to custom tuned profiles directory configurable or at least in subdirectory /etc/tuned/profiles, in order to mount them as a whole in containerized environments. 
+This will have to be implemented on [Tuned](https://github.com/redhat-performance/tuned) - [RHEL-26157](https://issues.redhat.com/browse/RHEL-26157)
+
+* Allow static key IPIs to be disabled (caused when sockets are opened with timestamping enabled). Currently these IPIs cause latency hits for low latency applications. This applies to RHEL and also to OpenShift deployments. Currently, to prevent these IPIs, tuned has an [rtentsk plugin](https://github.com/redhat-performance/tuned/blob/master/tuned/plugins/plugin_rtentsk.py) that opens a socket with timestamping enabled and holds this socket open.
+This will have to be implemented on RHEL + RHCOS - [RHEL-23928](https://issues.redhat.com/browse/RHEL-23928)
+
+### Workflow Description
```

Comment:
Not sure if it belongs here, but it would be nice to have a walkthrough of the steps for the initial deployment of an SNO with the NTO to show how everything gets set up and when/how the server is restarted and the new MC is applied to create the new systemd service.